jena-rdfconnection/src/main/java/org/apache/jena/rdflink/RDFLink.java

```diff
@@ -355,7 +354,9 @@ public default boolean queryAsk(Query query) {
      * @return QueryExecution
      */
     @Override
-    public QueryExec query(Query query);
+    default public QueryExec query(Query query) {
```

Comment:
I updated RDFLink's default shortcuts to delegate to `newQuery().query(...).build()` so that any processing that happens in QueryExecBuilder is applied by default.